{
  "level": "warn",
  "ts": "2022-11-12T10:34:47.433Z",
  "logger": "etcd-client",
  "caller": "v3/retry_interceptor.go:62",
  "msg": "retrying of unary invoker failed",
  "target": "etcd-endpoints://0xc000426700/127.0.0.1:2379",
  "attempt": 0,
  "error": "rpc error: code = FailedPrecondition desc = etcdserver: root user does not exist"
}